helpers/deploymentConfig.ts

```diff
@@ -188,6 +202,14 @@ export const chainlinkFeed: Config = {
     crvUSD: "0xEEf0C605546958c1f899b6fB336C20671f9cD49F",
     stETH: "0xCfE54B5cD566aB89272946F602D76Ea879CAb4a8",
   },
+  arbitrumsepolia: {
+    WBTC: "0xFb8d93FD3Cf18386a5564bb5619cD1FdB130dF7D",
+    USDC: "0x86f096B1D970990091319835faF3Ee011708eAe8",
+    USDT: "0xf3118a17863996B9F2A073c9A66Faaa664355cf8",
+    ARB: "0x4371bb358aB5cC192E481543417D2F67b8781731",
+    WETH: "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73",
+    XVS: "0x47fA6E9F717c9eB081c4734FfB5a1EcD70508891",
+  },
```

Comment:
These shouldn't be the token addresses, but the Chainlink feed addresses

```suggestion
  arbitrumsepolia: {
    WBTC: "0x56a43EB56Da12C0dc1D972ACb089c06a5dEF8e69",
    USDC: "0x0153002d20B96532C639313c2d54c3dA09109309",
    USDT: "0x80EDee6f667eCc9f63a0a6f55578F870651f06A4",
    ARB: "0xD1092a65338d049DB68D7Be6bD89d17a0929945e",
    WETH: "0xd30e2101a97dcbAeBCBC04F14C3f624E67A35165",
  },
```

Reply:
Got it. https://github.com/VenusProtocol/oracle/pull/174/commits/5f4f8be3d496f4e7df1ad32a54a4aa6acf5ee422